src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DIVIDE_CHARACTER = ",";
+    private static final String FIRST_COLUMN_WORD = "supply";
+    private static final int FIRST_COLUMN_INDEX = 0;
+    private static final int SECOND_COLUMN_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String line = reader.readLine();
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/156338484/73dc4bf2-27fd-4b57-a687-614050ca57b0)
Lets make this structure